src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -584,6 +605,30 @@ public function iShouldBeAbleToSelectPaymentMethod(string $paymentMethodName): v
         Assert::notFalse(array_search($paymentMethodName, array_column($paymentMethods, 'name'), true));
     }
 
+    /**
+     * @Then I should see :firstPaymentMethodName and :secondPaymentMethodName payment methods
+     */
+    public function iShouldSeeAndPaymentMethods(string $firstPaymentMethodName, string $secondPaymentMethodName): void
```

Comment:
You can use here `...` operator:
```suggestion
    public function iShouldSeeAndPaymentMethods(string ...$paymentMethodsNames): void
```